src/controllers/moviesController.js

```
@@ -0,0 +1,40 @@
+const { statusCode } = require("../helpers/constants.js");
+const {
+  getMovies,
+  getMovieById,
+} = require("../services/moviesService.js");
+
+const getAllMovies = async (req, res, next) => {
+  try {
+    const movies = await getMovies();
+
+    res.status(statusCode.OK).send(movies);
+  } catch (err) {
+    next(err)
+  }
+};
+
+
+const getMovie = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const movieById = await getMovieById(id);
+
+    if (!movieById) {

```

check comment abowe, you can just throw an error